src/akkudoktoreos/server/eos.py

```diff
@@ -912,28 +1028,25 @@ def site_map() -> RedirectResponse:
 
 # Keep the proxy last to handle all requests that are not taken by the Rest API.
 
-if config_eos.server.startup_eosdash:
```

Comment:
did it break the proxy?

Reply:
> did it break the proxy?

Proxy operation is not related to startup_eosdash. EOSdash may be started by other means.